test/integration_test/migration_test.go

```diff
@@ -1,4 +1,4 @@
-// +build integrationtest
+// +buil integrationtest
```

Comment:
looks like a typo ? - the build is failing as well

Reply:
yes, this was a typo. Fixed.